scripts/signup.py

```diff
@@ -17,15 +18,30 @@ def signup(args):
     else:
         print(response.text)
 
+def register_producer(args):
```

Comment:
Use full argument list instead of argparse structure: the function may be called from proof producer daemon

Reply:
Sure, also changed for signup